From 0485a4258cbbc7d2ba29790498c38f3baae43245 Mon Sep 17 00:00:00 2001
From: "yi.zhang1" <yi.zhang1@amlogic.com>
Date: Tue, 28 Nov 2023 17:00:45 +0800
Subject: [PATCH 8/9] transcoding: README for transcodinf demo [1/1]

PD#SWPL-144371

Problem:
README for transcoding demo.

Solution:
README for transcoding demo.

Verify:
local

Signed-off-by: yi.zhang1 <yi.zhang1@amlogic.com>
Change-Id: Ib8230271936d35dd7c3217bd66515c157662d823
---
 videotranscoding-demo/README | 28 ++++++++++++++++++++++++++++
 1 file changed, 28 insertions(+)
 create mode 100644 videotranscoding-demo/README

diff --git a/videotranscoding-demo/README b/videotranscoding-demo/README
new file mode 100644
index 0000000..1f724c5
--- /dev/null
+++ b/videotranscoding-demo/README
@@ -0,0 +1,28 @@
+Description:
+videotranscoding_demo is an demo for convert to different resolutions,codec and fps.
+
+How to use:
+--------------
+videotranscoding_demo <media-file> <config-file>
+
+Pass in two parameters, the first parameter is the converted multimedia file, and the second parameter is the configuration file.
+
+The configuration file format is as follows
+
+src_size={1920, 1080}
+src_codec=MPEG4
+src_framerate=60
+dst_size={720, 480}
+dst_codec=H264
+dst_framerate=30
+bitrate_kb=3000
+gop_size=60
+
+src_size is the original video resolution.
+src_codec is the original video encoding format.
+src_framerate is the original video fps.
+dst_size is the video resolution you want to convert to.
+dst_codec is the video encoding format you want to convert to.
+dst_framerate is the video fps you want to convert to.
+bitrate_kb is the code rate of encoder.
+gop_size is the GOP size.
-- 
2.25.1

